### PR TITLE
Feature/second print test

### PR DESCRIPTION
**Pull request by NicoLeNeko**

### CODE REVIEW

File: test.py [hunk 1]
```diff
@@ -1 +1,3 @@
+# Github comment
```
Comment (duchainer) [inline test.py:1]: Make comment more explicit

File: test.py [hunk 2]
```diff
@@ -1 +1,3 @@
+# Github comment 
 print("test!!")
+print("test number2!!")
```
Comment (duchainer) [inline test.py]: Number 2 should have a space between "number" and "2"

**Review by duchainer — CHANGES_REQUESTED** [review]
Good, just change the 3rd line like explained

File: test.py (hunk 1 above)
Comment (NicoLeNeko) [reply test.py:1]: oki

**Review by duchainer — APPROVED** [review]
Good to go

**Pull request merged**